alodataset/crowd_human_dataset.py

```diff
@@ -39,21 +39,18 @@ def __init__(
             assert ann_file is not None or "test" in img_folder, "When sample = False and the test split is not used, ann_file must be given."
 
         if "test" in img_folder:
-            self.dataset_dir="/data/crowdhuman/"
+            if not os.path.exists(os.path.join(self.dataset_dir, img_folder)):
+                self.dataset_dir = input("You are trying to use the test split of CrowdHuman. Please enter the path to the dataset directory: ")
```

Comment:
I think you can remove this as it's already implemented in BaseDatase [here](https://github.com/Visual-Behavior/aloception-oss/blob/master/alodataset/base_dataset.py#L157)